### ISSUE TITLE
Incorrect zoom center

### ISSUE THREAD

**Issue by Kemzi-coder** [openspacelabs/react-native-zoomable-view#54]
When using the zoomTo method or double-tapping with doubleTapZoomToCenter prop set to true, the ZoomableView zooms to the upper left corner instead of the center of the View. Because of this, the animation is jumping during the resetting of the zoom level if the bindToBorders prop is set to true.

Snack: https://snack.expo.dev/xUVUPAx-Z

**Comment by DLkhazan** [comment]
Having the same issue, did you find anything useful? 
@nicholas-patterson @Kemzi-coder

**Comment by DLkhazan** [comment]
@thomasttvo can you provide any input on this?

**Comment by 20chad16** [comment]
I am running into this issue when having two zoomable views side-by-side.  Has anyone found a solution or workaround?

@thomasttvo

**Comment by DLkhazan** [comment]
@20chad16 you need to disable panning from the node_module.

**Comment by c00ler82** [comment]
@DLkhazan Can you elaborate? I'm running at the issue, and I'm not an expert with animation.

**Comment by DLkhazan** [comment]
I can provide you the file.
Navigate to APP_FOLDER/node_modules/@openspacelabs/react-native-zoomable-view/src/ReactNativeZoomableView.tsx. And replace it with the attached file.

I made multiple changes to the file but I think the one you required will be achieved by modifying _handleShifting function. You can compare the library and modified function from the file. 
had to compress it, because they won't let me post it in original formate. But let me know if you need my help to understand it.
[zoomableview.zip](https://github.com/openspacelabs/react-native-zoomable-view/files/12142223/zoomableview.zip)

**Comment by c00ler82** [comment]
Yeah probably _handleShifting what I was missing. Thank you @DLkhazan. Worked like a charm.
